wls-basisdaten-service/src/main/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/domain/referendumvorlagen/Referendumoption.java

```diff
@@ -0,0 +1,31 @@
+package de.muenchen.oss.wahllokalsystem.basisdatenservice.domain.referendumvorlagen;
+
+import jakarta.persistence.Embeddable;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Embeddable
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString(onlyExplicitlyIncluded = true)
```

Comment:
Was ist der Mehrwert dieser Annotation, wenn alle Properties unten inkludiert sind?

Reply:
das neue Properties bewusst hinzugefügt werden sollen. Es soll vermieden werden dass ungewollt via toString ein Loading oder eine inifity-loop entsteht.

Reply:
Inspiration war dieser Artikel: https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/#3-lombok-annotations-you-need-to-avoid